assets/src/js/Components/ArchivePicker/SingleSidebar.js

```diff
@@ -76,12 +76,11 @@ export default function SingleSidebar({image}) {
         </div>
         <button
           className="close-sidebar"
+          aria-label={__('Close', 'planet4-master-theme-backend')}
```

Comment:
Since we set the `aria-label` property then we don't need to define a "Close" text and also remove the `text-indent` property [from here](https://github.com/greenpeace/planet4-master-theme/pull/2181/files#diff-61826b7e198488e0d8bd10ca420f37ebf03d0241dff3417b92560675f2142451L335).